### PR TITLE
[AIRFLOW-XXX] Docs improvements

### PR DESCRIPTION
**Pull request by mik-laj**
Make sure you have checked _all_ steps below.

### Jira

Not applicable

### Description

There are 8 more warnings left.
```build succeeded, 8 warnings.```

/CC: @ashb 

### Tests

Not applicable

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

File: CHANGELOG.txt [hunk 1]
```diff
@@ -941,7 +941,7 @@ Airflow 1.10.0, 2018-08-03
 - [AIRFLOW-1688] Support load.time_partitioning in bigquery_hook
 - [AIRFLOW-1975] Make TriggerDagRunOperator callback optional
 - [AIRFLOW-1480] Render template attributes for ExternalTaskSensor fields
-- [AIRFLOW-1958] Add **kwargs to send_email
```
Comment (ashb) [inline CHANGELOG.txt]: This isn't the right fix - we need to change how we pull in CHANGELOG to RST instead of needing to have a .txt file be vailid .rst

Comment (mik-laj) [reply CHANGELOG.txt]: I reverted this change. In fact, the problem seems to be bigger and should be done with a separate PR.

The rest of the changes is correct?

**Comment by kaxil** [comment]
Nice work @mik-laj . What are the other 8 warnings, can we not fix that in this PR?

**Comment by mik-laj** [comment]
List of current warning: 
````
1. ../CHANGELOG.txt:944: WARNING: Inline strong start-string without end-string.
2. ../CHANGELOG.txt:2170: WARNING: Inline emphasis start-string without end-string.
3. ../CHANGELOG.txt:2170: WARNING: Inline strong start-string without end-string.
4. /workspace/docs/code.rst:326: WARNING: Inline literal start-string without end-string.
5. /workspace/docs/code.rst:330: WARNING: Inline literal start-string without end-string.
6. WARNING: autodoc: failed to import class 'mesos_executor.MesosExecutor' from module 'airflow.contrib.executors'; the following exception was raised:
cannot import name 'LoginMixin'
7. /workspace/docs/howto/operator.rst:19: WARNING: Duplicate explicit target name: "google cloud functions api documentation".
8. /workspace/docs/integration.rst:188: WARNING: undefined label: azurecosmosinsertdocumentoperator (if the link has no caption the label must precede a section header)
9. /workspace/docs/integration.rst:189: WARNING: undefined label: azurecosmosdocumentsensor (if the link has no caption the label must precede a section header)
10. WARNING: html_static_path entry '/workspace/docs/_static' does not exist
````
Warnings with numbers 1 to 3 must be repaired in a separate PR. (Discussion: https://github.com/apache/airflow/pull/4588#discussion_r251004599). I do not plan to fix it for this reason in this PR.
Warning with numbers 4 to 5 I do not understand the reason why this message occurs. I would like to fix it, but if someone helps me, I will be happy.
Warning with numbers 6. This is a code problem that does not work. Because of this warning, our friend today wrote a message on the devlist of the future of Mesos (Discussion: https://lists.apache.org/list.html?dev@airflow.apache.org). I do not plan to fix it for this reason.
Warning with numbers 7 to 9. I plan to fix it when I find the moments. I have not found the reason for these warning yet. I would be happy if someone looked at it.
Warning with number 10. I did not look at it. 
There is still a warning with number 11, but it is not displayed. It is possible that when we correct the previous one, it will repair itself.

Summary:
3 seperate PR
4 need help/time
1 paused for discussions about the future
2 Not analyzed

In my opinion, the PR should be merged. Each warning are completely independent of each other. There is no change that is applied partially. Small steps forward. Yesterday no one thought to check the documentation automatically. Today is a small step away from us.

**Comment by holdenk** [comment]
Hi @mik-lag, I was looking at the code.rst error, and I noticed on the line above each of the errors the value  None was written as:

> ``None`

but I think this might be continuing to the next line causing the misleading error message and it might work if it is written as:

> \`\`None\`\`

I'm really happy to see someone working on improving the docs so thanks a lot :)

**Comment by mik-laj** [comment]
Hello,

Thank you for showing me this place  I have seen it many times and have not seen it. This is what I was looking for. I fixed other problems also now.

Best regards, 

# Current situation:

````
build succeeded, 4 warnings.
````

````
../CHANGELOG.txt:944: WARNING: Inline strong start-string without end-string.
../CHANGELOG.txt:2170: WARNING: Inline emphasis start-string without end-string.
../CHANGELOG.txt:2170: WARNING: Inline strong start-string without end-string.
````
This is not a change in the documentation, but we have to write some code and therefore a ticket number is needed.
```
WARNING: autodoc: failed to import class 'mesos_executor.MesosExecutor' from module 'airflow.contrib.executors'; the following exception was raised:
cannot import name 'LoginMixin'
```
This is not a documentation problem. A change in the source code is required.


@ashb @kaxil Can I ask you to review the changes and merge them?

**Review by ashb — APPROVED** [review]

**Comment by ashb** [comment]
I'm not sure how to fix the CHANGELOG one - maybe we do have make it valid markdown or RST at least.

**Comment by mik-laj** [comment]
I also think that it is worth to make a valid RST document.  It is worth pointing out that it is currently lan RST-ike document, because it contains headers in this format.

**Pull request merged**